src/serverMiddleware/proxy.js

```diff
@@ -30,8 +30,9 @@ console.log(
 console.log("PROXY TARGET:", proxyOptions.target);
 
 const vueRoutes = require("./routes.js");
-const isNuxtRoute = (url) =>
-	vueRoutes.some((regexString) => !!new RegExp(regexString).exec(url));
+const isNuxtRoute = (url) => {
+	return vueRoutes.some((regexString) => !!new RegExp(regexString).exec(url));
+};
```

Comment:
```suggestion
const isNuxtRoute = (url) => 
	vueRoutes.some((regexString) => !!new RegExp(regexString).exec(url));
```
hier hat sich inhaltlich nichts geändert, dann kann auch der code so bleiben wie er ist :D